code/modules/admin/verbs/adminhelp.dm

```diff
@@ -107,42 +113,50 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
 							continue
 			msg += "[original_word] "
 
-	if(!mob)	return	//this doesn't happen
+	if(!mob)	
+		return
 
-	var/mentor_msg = "<br><br><font color='#009900'><b>[selected_upper]: [get_options_bar(mob, 0, 0, 1, 0)]:</b></font> <br><font color='#DA6200'><b>[msg]</font></b><br>"
+	var/mentor_msg = "<br><br><font color='#009900'><b>[selected_upper]: [get_options_bar(mob, 1, 1, 1, 0)]:</b></font> <br><font color='#DA6200'><b>[msg]</font></b><br>"
 	msg = "<br><br><font color='#009900'><b>[selected_upper]: [get_options_bar(mob, 2, 1, 1)]:</b></font> <br><font color='#DA6200'><b>[msg]</font></b><br>"
 
 	var/admin_number_afk = 0
 
 	var/list/mentorholders = list()
-	var/list/debugholders = list()
 	var/list/adminholders = list()
 	for(var/client/X in admins)
-		if(R_MENTOR & X.holder.rights && !(R_ADMIN & X.holder.rights)) // we don't want to count admins twice. This list should be JUST mentors
+		if((R_MENTOR & X.holder.rights) && (!(R_ADMIN & X.holder.rights) || !(R_MOD & X.holder.rights))) // we don't want to count admins twice. This list should be JUST mentors
```

Comment:
```suggestion
		if((R_MENTOR & X.holder.rights) && !(R_ADMIN & X.holder.rights) && !(R_MOD & X.holder.rights)) // we don't want to count admins twice. This list should be JUST mentors
```
The negation of `A || B` is either `!A && !B` or `!(A || B)`, not `!A || !B`
Choose whichever you find more clear.